frontend/public/components/utils/file-input.tsx

```diff
@@ -42,11 +42,13 @@ class FileInputWithTranslation extends React.Component<FileInputProps, FileInput
     let fileIsBinary = false;
     const reader = new FileReader();
     reader.onload = () => {
-      const input = reader.result as string;
+      const input = fileIsBinary
+        ? (reader.result as string).split(',')[1]
+        : (reader.result as string);
       // OnLoad, if inputFileIsBinary we have read as a binary string, skip next block
       if (containsNonPrintableCharacters(input) && !fileIsBinary) {
         fileIsBinary = true;
-        reader.readAsBinaryString(file);
+        reader.readAsDataURL(file);
```

Comment:
We'll need a Bugzilla bug for this and backport the fix.

The issue seems to be that we should only use `Base64.encode` on Unicode strings. `btoa` should be used instead of binary data. It's a little strange, but I'm OK using `readAsDataURL` and stripping the prefix.

https://github.com/dankogai/js-base64#decode-vs-atob-and-encode-vs-btoa